examples/jku_backends.py

```diff
@@ -24,7 +23,7 @@
 print(jku_backend.status())
 
 # returns the provider of the backend
-print(jku_backend.provider)
+print(jku_backend.provider())
```

Comment:
i think this is wrong with the provider is not a function. @diego-plan9 is this correct.

Reply:
It is currently a function. See [here](https://github.com/Qiskit/qiskit-terra/blob/88a15d49ee094afdf731c27a8ee7fc9d768c7297/qiskit/providers/basebackend.py#L63).

Reply:
ok my bad

Reply:
yes, basically comes from https://github.com/Qiskit/qiskit-terra/pull/1312